drivers/espi/espi_npcx.c

```diff
@@ -865,6 +865,14 @@ static int espi_npcx_init(const struct device *dev)
 		return ret;
 	}
 
+	if (IS_ENABLED(CONFIG_ESPI_NPCX_BYPASS_CH_ENABLE_FATAL_ERROR)) {
+		/* Enable the access to the NPCX_ONLY_ESPI_REG2 register */
+		inst->NPCX_ONLY_ESPI_REG1 = 0x55;
```

Comment:
Prefer this doesn't use a magic number.  I assume 0x55 is a special value that needs to be written.  How about 
```
#define NPCX_ONLY_ESPI_REG1_UNLOCK_REG2 0x55
#define NPCX_ONLY_ESPI_REG1_LOCK_REG2 0
```

Reply:
Done.